docs/content/sdks/golang/build-transaction.md

```diff
@@ -3,14 +3,14 @@ title: Build a Transaction
 description: How to prepare a transaction with the Flow Go SDK
 ---
 
-Flow, like most blockchains, allows users to submit transactions that mutate
+Flow, like most blockchains, allows anybody to submit a transaction that mutates
 the shared global chain state. A transaction is an object that holds a payload,
 which describes the state mutation, and one or more authorizations that permit 
 the transaction to mutate the state owned by specific accounts.
 
 You can read more about transactions in the [transaction lifecycle documentation](/concepts/transaction-lifecycle).
 
-## Create the Transaction
+## Create the transaction
```

Comment:
Erm, title case here should be "Create the Transaction"? Is this an AP rule?

Reply:
Ah yep I'll fix all the casing

Reply:
I updated this.